token/cli/src/main.rs

```diff
@@ -3586,6 +3624,41 @@ fn app<'a, 'b>(
                 )
                 .arg(mint_decimals_arg())
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::WithdrawExcessLamports.into())
+                .about("Withdraw lamports from a Token Program owned account")
+                .arg(
+                    Arg::with_name("source_account")
```

Comment:
nit for consistency with transfer
```suggestion
                    Arg::with_name("from")
```